trulens_eval/examples/quickstart/langchain_quickstart.ipynb

```diff
@@ -189,7 +189,7 @@
     "# Define a groundedness feedback function\n",
     "f_groundedness = (\n",
     "    Feedback(grounded.groundedness_measure_with_cot_reasons)\n",
-    "    .on(Select.RecordCalls.first.invoke.rets.context)\n",
+    "    .on(TruChain.select_context().collect())\n",
```

Comment:
why does this one use `...collect()` and the next not, even though the old versions of both are the same?

Reply:
`.collect()` is used fo groundedness to return all of the context here as an array. This is needed for groundedness because we need to identify supporting evidence from all of the context available.

This contrasts with context_relevance where we want to evaluate the relevance of each chunk individually.